k-distribution/include/builtin/domains.k

```diff
@@ -273,7 +273,7 @@ module COLLECTIONS
 
   syntax List ::= Set2List(Set) [function, hook(SET.set2list)]
   syntax Set ::= List2Set(List) [function, functional, hook(SET.list2set)]
-
+  rule List2Set(ListItem(E) LS) => SetItem(E)
```

Comment:
I don't understand this rule. Could you explain?

Reply:
@daejunpark it's as confusing to me as it is to you. Without this line, the function `List2Set(ListItem(1))` remains unevaluated. Remove the line then build K then run `make -j6` in k-distribution/tests/builtins/collections to reproduce.

Reply:
what if you replace `LS` with `.List`?

Reply:
No, with `.List` it does not work.